simulate

```diff
@@ -26,15 +29,22 @@ def run_cli():
                         choices=[UNIDIRECTIONAL_RING, BIDIRECTIONAL_RING, COMPLETE_GRAPH, ARBITRARY_GRAPH],
                         help='Simulate the automaton operating on GRAPH')
     parser.add_argument('-n', '--network-size',
-                        help="For UnidirectionalRing and BidirectionalRing and CompleteGraph must specify network size")
+                        help="For Uni/BidirectionalRing and CompleteGraph must specify network size")
     parser.add_argument('-gf', '--graph-file',
                         help='For Arbitrary graph, give a graph file definition')
     parser.add_argument('-t', '--type',
-                        choices=[SYNC_SIMULATOR, SYNC_SIMULATOR_WITH_UID, ASYNC_SIMULATOR_WITH_UID],
-                        help='type of simulator(Sync, Sync-UID, Async-UID)'
-                        )
+                        choices=[SYNC, ASYNC, ORDERED_ASYNC],
+                        help='type of simulator(sync, async, ordered-async)')
+    parser.add_argument('-r', '--random-ids', action='store_true',
+                        help="Generate random uids for graph nodes")
     parser.add_argument('-o', '--output', type=argparse.FileType('w'), default='-',
                         help='Output file to write result to it.')
+    parser.add_argument('--link-failure', metavar='PROBABILITY', default=0, type=float,
+                        help="Probability of message lost on every link. Float number between 0 and 1."
+                             " Under link-failure the default limit steps is " + str(DEFAULT_LIMIT))
+    parser.add_argument('-l', '--limit', type=int,
+                        help="Run simulator for LIMIT steps. Under link or node failure (stopping and byzantine) the "
+                             "default limit is " + str(DEFAULT_LIMIT) + " steps.")
```

Comment:
It's better to use `f"default limit is {DEFAULT_LIMIT} steps"` or `"default limit is {} steps".format(DEFAULT_LIMIT)` instead of change type of int to string and concat using `+`.